tensorflow/contrib/rnn/python/ops/gru_ops.py

```diff
@@ -129,13 +130,21 @@ class GRUBlockCell(rnn_cell_impl.RNNCell):
 
   """
 
-  def __init__(self, cell_size):
+  @deprecated_args(None, "cell_size is deprecated, use num_units instead",
+                   "cell_size")
+  def __init__(self, num_units=None, cell_size=None):
     """Initialize the Block GRU cell.
 
     Args:
-      cell_size: int, GRU cell size.
```

Comment:
must keep the old arg in the order it shows up in the signature; just say it's deprecated and means the same thing as num_units.

add a Raises: section with entry "ValueError: if both cell_size and num_units are not None; or both are None."

Reply:
Thanks @ebrevdo. The docstring has been updated.